root/layout/components/Footer.js

```diff
@@ -1,95 +1,520 @@
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('Donate')}
+                </a>
+              </li>
+              <li>
+                <a
+                  className="fw-bold fs-5"
+                  href="https://wiki.musicbrainz.org/Main_Page"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('Wiki')}
+                </a>
+              </li>
+              <li>
+                <a
+                  className="fw-bold fs-5"
+                  href="https://community.metabrainz.org/"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('Community')}
+                </a>
+              </li>
+              <li>
+                <a
+                  className="fw-bold fs-5"
+                  href="https://blog.metabrainz.org/"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('Blog')}
+                </a>
+              </li>
+              <li>
+                <a
+                  className="fw-bold fs-5"
+                  href="https://www.redbubble.com/people/metabrainz/shop"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('Shop')}
+                </a>
+              </li>
+              <li>
+                <a
+                  className="fw-bold fs-5"
+                  href="https://metabrainz.org/"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('MetaBrainz')}
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <div className="row mt-4">
+          <div className="col-md-3 border-top pt-4 d-none d-md-block fs-5">
+            <p>
+              {l('OSS Geek?')}
+              {' '}
+              <a
+                href="https://github.com/metabrainz/musicbrainz-server"
+                rel="noopener noreferrer"
+                target="_blank"
+              >
+                <span>
+                  {l('Contribute Here')}
+                </span>
+              </a>
+            </p>
+          </div>
+          <div className="col-md-6 border-top pt-4 text-center fs-5">
+            {
+              // eslint-disable-next-line max-len
+              exp.l('Brought to you by <span id="meb-logo"/> MetaBrainz Foundation')
+            }
+          </div>
+          <div className="col-md-3 border-top pt-4 d-none d-md-block fs-5">
+            <p>
+              {l('Found an Issue?')}
+              <a
+                className="ms-1"
+                href="https://tickets.metabrainz.org/"
+                rel="noopener noreferrer"
+                target="_blank"
+              >
+                <span>
+                  {l('Report Here')}
```

Comment:
"Report it here" (I'd also not use All Caps for this, but I can live with it if you feel it's important I guess)

Reply:
Right